app/models/user.rb

```diff
@@ -5,6 +5,9 @@ class User < ApplicationRecord
   has_paper_trail
   devise :database_authenticatable, :invitable, :recoverable, :rememberable, :validatable
 
+  validates :email, presence: true
```

Comment:
Looks good, but this looks unrelated to "bumping jquery to fix select2". Is there another issue this is solving?

Reply:
Hmm, odd. These are commits from #726; this PR was for the bump in `package.json` and `yarn.lock`